package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "townsquare",
-  "version": "2.18.2",
+  "version": "2.18.3",
   "description": "Blood on the Clocktower Town Square",
   "author": "Steffen Baumgart",
   "scripts": {
-    "clone-images": "rm -rf ./tmp/botc-icons/ && git clone https://github.com/tomozbot/botc-icons.git --depth 1 ./tmp/botc-icons",
-    "copy-images": "cp -rf ./tmp/botc-icons/WEBP/ ./src/assets/icons/",
+    "clone-images": "rimraf ./tmp/botc-icons/ && git clone https://github.com/tomozbot/botc-icons.git --depth 1 ./tmp/botc-icons",
+    "copy-images": "copyfiles -u 3 ./tmp/botc-icons/WEBP/* ./src/assets/icons/",
```

Comment:
Can you revert these?   These commands don't exist on Linux, and I don't want to install them.

Reply:
Oh, I see they are node commands.  Any reason to use them over native commands?